index.js

```diff
@@ -1,22 +1,35 @@
 import 'babel-polyfill';
 import express from 'express';
+import cors from 'cors';
+import env from 'dotenv';
+import volleyLogger from 'volleyball';
+import expressValidator from 'express-validator';
 import swaggerUi from 'swagger-ui-express';
 import YAML from 'yamljs';
 import routes from './server/routes';
 
+env.config();
 const swaggerDocument = YAML.load('swagger.yaml');
 const app = express();
+
+app.use(cors());
+app.use(volleyLogger);
+app.use(expressValidator());
+app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
+
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
 app.use('/api/v1', routes);
 
 const port = process.env.PORT || 3000;
 
-app.listen(port, () => {
+if (!module.parent) {
```

Comment:
What's the conditional for?

Reply:
I already remove it because it affected the test coverage. It is just to prevent `Uncaught Error: listen EADDRINUSE: address already in use :::4000` that may arise when trying to start the server when it's already running